python/PyQt6/analysis/auto_generated/interpolation/qgsgridfilewriter.sip.in

```diff
@@ -42,22 +42,22 @@ cancellation support
 :return: 0 in case of success
 %End
 
-    void setCreateOptions( const QStringList &list );
+    void setCreationOptions( const QStringList &list );
```

Comment:
for readability
```suggestion
    void setCreationOptions( const QStringList &creationOptions );
```

Reply:
Maybe better just `options` as there's already the method named `creationOptions()` ?

Reply:
> Maybe better just `options` as there's already the method named `creationOptions()` ?

Yeah, that works too, no strong opinions.